x-pack/test/smb-fixture/src/main/java/org/elasticsearch/test/fixtures/smb/SmbTestContainer.java

```diff
@@ -7,28 +7,34 @@
 
 package org.elasticsearch.test.fixtures.smb;
 
+import com.github.dockerjava.api.model.Capability;
+
 import org.elasticsearch.test.fixtures.testcontainers.DockerEnvironmentAwareTestContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.containers.wait.strategy.WaitAllStrategy;
 import org.testcontainers.images.builder.ImageFromDockerfile;
 
+import java.time.Duration;
+
 public final class SmbTestContainer extends DockerEnvironmentAwareTestContainer {
 
-    private static final String DOCKER_BASE_IMAGE = "ubuntu:16.04";
+    private static final String DOCKER_BASE_IMAGE = "ubuntu:24.04";
```

Comment:
There is a problem with installation of Samba in Ubuntu 16.04 on specific Docker configuration. The installation hangs on some Python modules compilation.
Python modules compilation hangs because in RHEL9-based distros, during Docker build, `ulimit` set to `unlimited`. This is hard to change so instead I've decided to upgrade Ubuntu image to avoid this problem.

Reply:
awesome!